atom/common/javascript_bindings.cc

```diff
@@ -145,8 +145,10 @@ void JavascriptBindings::IPCSendShared(mate::Arguments* args,
   if (!is_valid() || !render_frame())
     return;
 
-  base::SharedMemoryHandle memory_handle = shared_memory->handle().Duplicate();
+  base::SharedMemoryHandle memory_handle =
+      base::SharedMemory::DuplicateHandle(shared_memory->handle());
```

Comment:
base::SharedMemory::DuplicateHandle seems also calling handle.Duplicate(). Why requires changes?

Reply:
most everything else seemed to be using the static method so I changed just to match